### ISSUE TITLE
Analyze work but applying does nothing

### ISSUE THREAD

**Issue by jrmgx** [Sappharad/mp3gainOSX#30]
Hello and thank you for that piece of software.
I'm using it on some m4a and when I ask for an analysis, it seems to work
it returns values that even seems right (regarding the state of those files)

<img width="627" alt="Capture d’écran 2021-06-13 à 11 58 07" src="https://user-images.githubusercontent.com/42989/121802807-a8fb1d00-cc3e-11eb-9836-61c72a22a462.png">

_Pardon me about titles shown in this screen_

But when I click on "Apply gain" some similar process as analyze seems to happen (with a modal showing tracks being processed) but in the end the file is not changed (shasum give the same value for both files, previous one that I saved, and the one I put tried to change — also I can hear that no change has been made)

**Comment by Sappharad** [comment]
I just took a random .m4a file and tested this by boosting the volume to 99 dB and it worked fine. In your screenshot, you're setting the target volume to 89 dB but the files in your screenshot already very close to 89 dB. 

Although volumes are listed in fractional values, the precision of a volume change cannot actually go to a level that specific. (Example: 0.19) Since it's not actually re-encoding the file, the amount of volume change allowed depends on the input files. I don't actually know what the limit for AAC (M4A) is, but for MP3 I found a post here saying MP3Gain can only adjust in multiples of 1.5 dB.
https://hydrogenaud.io/index.php?topic=92033.0

In short, the adjustments in your screenshot are probably too small to actually make a difference on those files. Additionally, "Prevent Clipping" will limit your maximum depending on the file, but since none of your files are being increased in volume that does not matter here.

Please let me know if you think something is still wrong.
